docs/wiki/What-is-Enterprise-Scale.md

```diff
@@ -33,7 +33,11 @@ The following table outlines key customer requirements in terms of landing zones
 
 ## Pricing
 
-There’s no cost associated with Enterprise-Scale, and you only pay for the Azure services that are being enabled, and the services your organization will deploy into the landing zones. For example, you don’t pay for the management groups or the policies that are being assigned, but policy to enable Azure Security Center on the landing zone subscriptions will generate cost on those subscriptions.
+There’s no cost associated with Enterprise-Scale itself, as it is just an architecture that is constructed using existing Azure products and services. Therefore you only pay for the Azure products and services that you choose to enable, and also the products and services your organization will deploy into the various application/service landing zones for your workloads.
```

Comment:
can we remove 'various application/service' from the sentence, and just leave 'deploy into the landing zones'?

Reply:
sure just changed in latest commit 👍